ai-management/ai-management-kubernetes.yml

```diff
@@ -6,4 +6,6 @@ spring:
     hikari:
       maximum-pool-size: 5
       initial-pool-size: 1
-      connection-timeout: 20000
\ No newline at end of file
+      connection-timeout: 20000
+  kafka:
+    bootstrap-servers: b-1.agenatech-dev.9cindp.c2.kafka.eu-north-1.amazonaws.com:9094, b-2.agenatech-dev.9cindp.c2.kafka.eu-north-1.amazonaws.com:9094git push
```

Comment:
amazonaws.com:9094**git push** 

git push  should be removed :)

Reply:
Fixed